mozilla/foundation.mozilla.org
# [6439] Migrate update model to wagtail pages

Partially addresses #6439 - Part 1 of 2. This should be reviewed and merged before part 2 (Part 2 is in PR #6490 with most of the code cleanup)

Acceptance criteria:
- [x] Create a new Update model (copied exactly from the previous one) and make sure it lives in the `wagtailpages/` app. 
- [x] Create a data migration to copy the Orderables from the old model to the new model, and update the Orderables. 
  - [x] Make sure the orderables maintain their ordering. 
  - [x] May require creating a temporary field on the Orderable (called update_2 or something temporary) so we can hold both Updates at the same time, then delete the old field, and rename the new field. 

Steps for testing:
1. Edit a PNI Product
2. Add a Update (in the "updates" orderable section). Or maybe add like 5 updates. 
3. In `networkapi.wagtailpages.pagemodels.products` lines 304-306 add `SnippetChooserPanel('update_new'),` under `SnippetChooserPanel('update'),`  so you can see the old and the new updates. The new ones should all be blank.
4. Run `python manage.py migrate_product_updates` 
5. Refresh your edit view in the wagtail admin, the `update_new` and `update` Snippet Choosers should be identical now. But if you edit the top one it'll bring you the buyersguide snippet URL in the URL bar. And the bottom one should bring you to the wagtailpages snippet URL

**Note for production**:
We need to run `python manage.py migrate_product_updates` in production before step 2 is merged in.
